at_functional_test/test/config_verb_test.dart

```diff
@@ -79,15 +82,21 @@ void main() {
     await socket_writer(socketFirstAtsign!, 'config:block:add:@@kevin');
     var response = await read();
     print('config verb response : $response');
-    assert(response.contains('error:AT0003-Invalid syntax'));
+    response = response.replaceFirst('error:', '');
+    var decodedResponse = jsonDecode(response);
+    expect(decodedResponse['errorCode'], 'AT0003');
+    expect(decodedResponse['errorDescription'], contains('Invalid syntax'));
```

Comment:
Won't this (and other similar tests) fail against current prod version of the server? Perhaps you need to assert that either the old check response.contains(...) or the new check (decodedResponse.errorCode and errorDescription) succeeds?